sources/lists/mdl.go

```diff
@@ -1,116 +0,0 @@
-package lists
```

Comment:
I downloaded the blacklist and the most recent entry was from 2017 while the oldest was from 2009.